docs/usage/create.rst

```diff
@@ -74,3 +74,14 @@ Examples
     $ cd /home/user/Documents
     # The root directory of the archive will be "projectA"
     $ borg create /path/to/repo::daily-projectA-{now:%Y-%m-%d} projectA
+
+    # Use external command to determine files to archive
+    # Use --paths-from-stdin with find to only backup files less than 1MB in size
+    $ find ~ -size -1000k | borg create --paths-from-stdin /path/to/repo::small-files-only
+    # Use --paths-from-command with find to only backup files from a given user
+    $ borg create --paths-from-command -- /path/to/repo::joes-files find ~ -user joe
+    # Use --paths-from-stdin with --paths-separator to backup files listed in a csv file
+    $ echo files-to-backup.csv | borg create \
+        --paths-from-stdin \
+        --paths-separator "," \
+        /path/to/repo::csv-specified-files
```

Comment:
maybe csv is a bad example as this would expect a single row of comma separated filenames.
also, echo obviously does not work - please **actually try** the stuff you are adding as example.

maybe using NUL chars (0x00) as separator would be a good example (together with find which iirc has an option to output using that).

Reply:
Sorry about that. Guess that's what I get for not testing the *entire* example command. The find with NUL separator idea seems almost more contrived because you can just not use that option on find and not specify a separator with borg. Although I'm not sure CSV is a good example either, since when I tested it just now vim added a newline to the end that tripped things up.

Reply:
Honestly it's fairly hard to find a use case for that option that isn't somewhat contrived.

Reply:
From `find` man page:
```
       -print0
              True; print the full file name on the standard output,
              followed by a null character (instead of the newline
              character that -print uses).  This allows file names that
              contain newlines or other types of white space to be
              correctly interpreted by programs that process the find
              output.  This option corresponds to the -0 option of
              xargs.
```
This is kind of a safety feature: the \0 character is used as string terminator in C, so we can be 100% sure it is not part of a path/filename. For \n, it usually works, but we cannot be 100% sure.